### PR TITLE
Add first script

### PR DESCRIPTION
**Pull request by giorgiosironi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
-# scripts
-Common scripts to use in Libero projects builds
+Libero dummy API
```
Comment (thewilkybarkid) [inline README.md]: Needs updating.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+dist: trusty
+
+language: minimal
```
Comment (thewilkybarkid) [inline .travis.yml]: Can also do `bash`.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+dist: trusty
```
Comment (thewilkybarkid) [inline .travis.yml]: Needed?

**Comment by thewilkybarkid** [comment]
Shellcheck can't recognise that they're not executable? Would be useful to check that.

File: .travis.yml (hunk 3 above)
Comment (giorgiosironi) [reply .travis.yml]: Would be helpful to run on multiple systems to check portability, but if it can be omitted, it's a good base case.

**Review by thewilkybarkid — APPROVED** [review]

**Pull request merged**